google/cloud/bigquery/schema.py

```diff
@@ -417,6 +435,7 @@ def _key(self):
             self.description,
             self.fields,
             policy_tags,
+            self.timestamp_precision,
```

Comment:
if you decide to do an Enum subclass, this could look like: self.timestamp_precision.value